helpers/searchBarHelper.js

```diff
@@ -0,0 +1,17 @@
+export const resetOrSubmitSearch = (value, searchTerm, minChars) => {
```

Comment:
value and searchTerm seem synonomous. What's the difference? Can we add comments or change the name of these params.